ssl/s3_lib.c

```diff
@@ -9,8 +9,10 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include "e_os.h"
```

Comment:
On side note I wonder what's in s3_lib.c that is [that] OS-specific...

Reply:
I'll check this one too.